docs/tutorials/services/navigate-with-rover-base.md

```diff
@@ -0,0 +1,646 @@
+---
+title: "Navigate with a Rover Base"
+linkTitle: "Navigate with a Rover"
+type: "docs"
+description: "Introduction to using a rover base with the navigation service."
+image: "/tutorials/navigate-with-rover-base/leo-in-office.png"
+imageAlt: "The leo rover navigating in the Viam Lab."
+images: ["/tutorials/navigate-with-rover-base/leo-in-office.png"]
+tags:
+  [
+    "try viam",
+    "navigation",
+    "movement sensor",
+    "vision",
+    "motion",
+    "movement sensor",
+    "base",
+    "rover",
+    "motion planning",
+    "obstacles",
+    "waypoints",
+  ]
+authors: ["Sierra Guequierre"]
+languages: ["python", "go"]
+viamresources: ["navigation", "base", "movement_sensor"]
+level: "Intermediate"
+date: "2023-02-08"
+# updated: ""
+cost: "0"
+no_list: true
+weight: 4
+# SMEs: Ray Bjorkman, Fahmina
+---
+
+One key feature of Viam is [Navigation](/manage/data/), the stateful definition of Viam's [motion service](/services/motion/).
+
+Using Navigation, you can queue up user-defined waypoints and expect your robot to move to them in the order that you specify.
```

Comment:
Bit strange to say they can expect their robot to do that. It makes me immediately go - will it... not do that? 
```suggestion
Using Navigation, you can queue up user-defined waypoints and your robot willl move to them in the order that you specify.
```